cypress/basic-typescript/cypress/e2e/login.cy.ts

```diff
@@ -0,0 +1,16 @@
+describe('My Login Application', () => {
+  it('should login with valid credentials', () => {
+    const loginCredentials = Cypress.env('login')
```

Comment:
Also an option (probably preferred):

```suggestion
    const { username, password } = Cypress.env('login')
```